### ISSUE TITLE
`go.mod` containing one or more replace directives

### ISSUE THREAD

**Issue by smallworlnd** [edouardparis/lntop#42]
Hello,

Running `go install github.com/edouardparis/lntop@latest` gives me the following error:

```
go install github.com/edouardparis/lntop@latest: github.com/edouardparis/lntop@v0.2.0
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

Go version:
`go version go1.16.7 freebsd/amd64`

Is this related to the OS, or maybe there's something up with my Go installation?

**Comment by edouardparis** [comment]
Thank you for the issue !

Because of a bug on the etcd versioning, I added a replace in go.mod. https://github.com/edouardparis/lntop/commit/b6d121437bdc0b7e0aec47e6a72a574238dd94c9
see lightningnetwork/lnd#5624
I did not think It would break the go install command.
The replace will be removed by the lnd v0.14 release.
For the moment please use the other method: git clone and go install ./...

**Comment by epther** [comment]
I ran into the exact same error when trying to update lntop from v0.1.0 to v0.2.0 by running `go install github.com/edouardparis/lntop@latest`

I attempted to use the other method of "git clone and go install ./..." but another error occurs:

```
go install ./...
# runtime/cgo
gcc: error: unrecognized command line option '-marm'; did you mean '-fasm'?
```

Searched around and it looks like it may be a go cross-compiler issue? Is there something wrong with my go installation?

Go version: `go version go1.17 linux/arm`
Running RaspiBlitz v1.7.0

**Comment by barbarix01** [comment]
I got the same errors at the respective installation method:

```go install: github.com/edouardparis/lntop@latest (in github.com/edouardparis/lntop@v0.2.0):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```
and 
```
# runtime/cgo
gcc: error: unrecognized command line option '-marm'; did you mean '-fasm'?
```

**Comment by smallworlnd** [comment]
This issue appears fixed for me now, on `lnd` version 0.14.1-beta and `lntop` 0.3.1. Thanks!
